pyiceberg/table/__init__.py

```diff
@@ -4393,12 +4397,12 @@ def _readable_metrics_struct(bound_type: PrimitiveType) -> pa.StructType:
                     "spec_id": data_file.spec_id,
                     "record_count": data_file.record_count,
                     "file_size_in_bytes": data_file.file_size_in_bytes,
-                    "column_sizes": dict(data_file.column_sizes),
-                    "value_counts": dict(data_file.value_counts),
-                    "null_value_counts": dict(data_file.null_value_counts),
-                    "nan_value_counts": dict(data_file.nan_value_counts),
-                    "lower_bounds": dict(data_file.lower_bounds),
-                    "upper_bounds": dict(data_file.upper_bounds),
+                    "column_sizes": dict(data_file.column_sizes) if data_file.column_sizes is not None else None,
```

Comment:
I believe that using `dict(data_file.column_sizes or {})` will achieve the same behavior without the need for an if statement. What do you think?

Reply:
nit: push this logic to the line above
```
                column_sizes = data_file.column_sizes or {}
```

Reply:
also i wonder if the null/None behavior is due to the `.toPandas()` conversion when queried through Spark